Sources/TSCBasic/Path.swift

```diff
@@ -821,7 +888,15 @@ extension AbsolutePath {
     /// This method is strictly syntactic and does not access the file system
     /// in any way.
     public func contains(_ other: AbsolutePath) -> Bool {
+#if os(Windows)
+        return self._impl.string.withCString(encodedAs: UTF16.self) { lhs in
+            other._impl.string.withCString(encodedAs: UTF16.self) { rhs in
+                PathIsSameRootW(lhs, rhs)
```

Comment:
Perhaps I'm missing something, but this doesn't seem to be equivalent to the non-Windows branch... It doesn't ensure that `other` is a subpath of `self`, right?

Reply:
Yeah, this isn't correct :/ however, seems to allow getting through a bootstrap of s-p-m (which is what this change is really about).  I don't have a problem with replacing it with a different implementation, the goal here is to just get to the point where we can get some form of builds of s-p-m.

Reply:
Turns out this change is not required, lets just drop it.